.travis.yml

```diff
@@ -13,6 +13,8 @@ before_install:
 install:
   - wget -q https://raw.githubusercontent.com/qiime2/environment-files/master/latest/staging/qiime2-latest-py35-linux-conda.yml
   - conda env create -q -n test-env --file qiime2-latest-py35-linux-conda.yml
+  # update default install with local recipe:
+  - conda env update -q -n test-env --file ci/recipe/meta.yaml
```

Comment:
Once you drop this we can re-run travis.

Reply:
I was waiting for environment-files/staged to be updated to push the revert. Nice. Let's see...